builtin/mainmenu/tab_settings.lua

```diff
@@ -71,6 +71,24 @@ local function video_driver_fname_to_name(selected_driver)
 	return ""
 end
 
+local default_label = fgettext("Default")
```

Comment:
`fgettext("Default")` directly in the formspec can be translated and it'd be necessary to modify the handler too so that it'd work, therefore it's easier to have a label out the formspec for translating it.